src/hooks/useArrowKeyFocusManager.js

```diff
@@ -13,59 +19,67 @@ import CONST from '../CONST';
  * @param {Boolean} [config.shouldExcludeTextAreaNodes] – Whether arrow keys should have any effect when a TextArea node is focused
  * @returns {Array}
  */
-export default function useArrowKeyFocusManager({maxIndex, onFocusedIndexChange = () => {}, initialFocusedIndex = 0, disabledIndexes = [], shouldExcludeTextAreaNodes = true}) {
+export default function useArrowKeyFocusManager({
+    maxIndex,
+    onFocusedIndexChange = () => {},
+    initialFocusedIndex = 0,
+    disabledIndexes = EMPTY_ARRAY,
```

Comment:
*question:* If a non-empty array value is passed for `disabledIndexes`, will that be referentially stable?

Reply:
No, to make it stable it has to be memoized, Objects `{}` and Arrays `[]` aren't stable and always will produce a re-render if it is into the dependency array